### ISSUE TITLE
feat(ci): Have some sort of health checks on live instances

### ISSUE THREAD

**Issue by anna-parker** [loculus-project/loculus#2380]
We were unaware lapis was breaking sequence submission for multi-segmented pathogens on the demo-instance. We should think about adding some checks for this

**Comment by theosanderson** [comment]
We now have status.pathoplexus.org which is wired up to my phone and to the pathoplexus email - do we think we can close this?

**Comment by chaoran-chen** [comment]
Thanks for setting this up! For Pathoplexus, it looks good! Do you also have something for loculus.org (I remember you said that you added it somewhere?) and main.loculus.org?

**Comment by theosanderson** [comment]
We have something on the pathoplexus uptimerobot simply to let us know if `loculus.org` goes entirely down. Unlike the Pathoplexus status it does not check LAPIS, etc.

**Comment by theosanderson** [comment]
Oh, wait, yes, and that is indeed `loculus.org`, not `main.loculus.org`..

**Comment by corneliusroemer** [comment]
Yes, uptime sounds good enough, we can add extra endpoints to check if we find that the current ones are not enough.

I've also signed in to uptime on my phone. Thanks for sharing details just now
